tests/upgrade/postinstall.test.js

```
@@ -36,6 +36,158 @@ test('postinstall: fresh install (no data dir) exits 0 silently', () => {
   fs.rmSync(tmp, { recursive: true, force: true });
 });
 
+test('postinstall: isDevWorktree detects /.genie/worktrees/ paths', () => {
+  // require() the script — module.exports surfaces the helpers without
+  // running main() (require.main !== module guard).
+  const mod = require(POSTINSTALL);
```

ESLint

[error] 'require' is not defined.

(no-undef)